data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,450 @@ mission "Unfettered returning home"
+			action
+				set "unfettered: beaten the bugs"
+	npc disable
+		government "Hai (Unfettered Challenger)"
+		to spawn
+			"total ships" == 1
+			has "flagship model: Anomalocaris"
+			not "unfettered: fled with the relic"
+			"old reputation: Hai (Unfettered)" < -2000
+		personality entering heroic mute marked plunders target unconstrained
+		ship "Shield Beetle (Prototypes)" "Requiem"
+		conversation
+			`You have managed to defeat the Unfettered challenger. Now it's time to land and claim your reward.`
+			action
+				set "unfettered: beaten the bugs"
+	npc
+		government "Hai (Unfettered)"
+		to spawn
+			not "unfettered: fled with the relic"
+		ship "Lightning Bug (Unfettered Shipyards)" "Dungeon Master"
+		personality launching marked staying surveillance uninterested
+		on provoke
+			dialog `Mera hails you. "Stop! I am only here as an observer. If you destroy my ship, the deal is off!"`
+		on kill
+			fail
+			set "unfettered: outcast"
+			dialog `As the Lightning Bug is destroyed you receive a hail from the local authorities. "You completely disrespected the rules of the duel and killed our most prominent researcher. You will not be welcome here anymore!"`
+	npc
+		government "Hai (Unfettered)"
+		to spawn
+			has "unfettered: fled with the relic"
+		personality unconstrained nemesis launching heroic marked
+		ship "Sea Scorpion (Shredder)" "Hunter"
+	on visit
+		dialog "Either you have not defeated the Unfettered challenger in orbit yet, or you have broken the terms of the duel."
+	to complete
+		not "unfettered: fled with the relic"
+		# Needed to make sure the mission doesn't auto complete if no ships spawn.
+		has "unfettered: beaten the bugs"
+		has "flagship model: Anomalocaris"
+	on complete
+		clear "unfettered: beaten the bugs"
+		clear "old reputation: Hai (Unfettered)"
+		log "Managed to use an old Anomalocaris to beat some Unfettered challengers, earning respect with the Unfettered Hai so long as the ship remains in your possession."
+		conversation
+			`When you land in one piece, Mera approaches, looking quite surprised. "You really won! I saw everything, and the scans of the effects of these exotic weapons will be quite useful for my research. Was it hard for you?"`
+			choice
+				`	"It was fun."`
+					goto next
+				`	"This was easy!"`
+				`	"It was a challenge."`
+					goto challenge
+			`	"Don't get overconfident, Captain," she muses. "I am sure you know what happens to such people.`
+				goto next
+			label challenge
+			`	"Oh, so we almost got that ship back then? Maybe I should have participated in the fight after all.`
+			label next
+			`	"Well, congratulations, I did not think you had it in you. I will give you authorization to land, on the condition that you keep this relic from our past in one piece. Remember, your reputation with us is solely due to to that ship. If you somehow lose it because of incompetence on your own end, it would look pretty bad to us!"`
```

Comment:
```suggestion
			`	"Well, congratulations, I did not think you had it in you. I will give you authorization to land, on the condition that you keep this relic from our past in one piece. Remember, right now, your reputation with us is solely due to to that ship. If you somehow lose it because of incompetence on your own end, it would look pretty bad to us true Hai!"`
```
The rep is temporary. Also, adding a little fun at the end.